src/renderer/components/+cluster/cluster-overview.scss

```diff
@@ -7,7 +7,7 @@
   display: grid;
   grid-gap: $gridGap;
   grid-template-columns: repeat(auto-fit, minmax(500px, 1fr));
-  grid-template-rows: 1fr 1fr;
+  grid-template-rows: auto;
```

Comment:
Not sure it is a good solution. Just want to automatically expand `Warnings` and use all available space if all cluster metrics are hidden. 

<img width="1190" alt="Screen Shot 2021-02-16 at 1 35 48 PM" src="https://user-images.githubusercontent.com/38247153/108045624-1b773a00-705d-11eb-9790-02fc930f79e6.png">

Reply:
A better solution would be to name grid row areas and switch between them in JS.